### ISSUE TITLE
Current Members of Parliament should be in House of Represenatives

### ISSUE THREAD

**Issue by kaerumy** [Sinar/popit-issues#6]
Parliament has two, Senate and Representatives, so current Members of Parliament should be under  House of Representatives or Dewan Rakyat.

https://sinar-malaysia.popit.mysociety.org/organizations/536844f619ee29270d8a9f63

**Comment by sweemeng** [comment]
2 new organization?

**Comment by kaerumy** [comment]
Yes, Senate and House of Representatives, not Parliament is more accurate

**Comment by kaerumy** [comment]
In current import the should be put under House of Representatives (MPs)
